src/types.cc

```diff
@@ -308,7 +308,8 @@ namespace ReverseDbReg {
   using T = ReverseDb;
 
   an<T> make(const string &file) {
-    an<T> db = New<ReverseDb>(string(RimeGetUserDataDir()) +  "/" + file);
+    an<T> db = New<ReverseDb>(
```

Comment:
Is it the only necessary change to pass the compilation?

Reply:
Not the only one.
I got more errors after resolving the first. I'm commenting inline for every change.